site/public/js/grade-inquiry.js

```diff
@@ -141,25 +141,8 @@ function onGradeInquirySubmitClicked(button) {
                 if (json['status'] === 'success') {
                     const data = json['data'];
```

Comment:
We are using a lot less data here. Thus in the PHP code, we should be able to remove a lot of data which is returned in the JSON response.